sdv/multi_table/base.py

```diff
@@ -69,12 +69,22 @@ def _print(self, text='', **kwargs):
         if self.verbose:
             print(text, **kwargs)  # noqa: T001
 
+    def _check_metadata_updated(self):
+        self.metadata._check_updated_flag()
+        if self.metadata._updated:
```

Comment:
Not sure if you need to store the `_updated` flag if you can just directly call `_check_metadata_updated`

Reply:
Yes, I did it to be consistent between single and multi-table metadata. I removed it in [d20f32a](https://github.com/sdv-dev/SDV/pull/1786/commits/d20f32acf9b84ca2c86522cd6217e0e52c711a9a)